include/nil/crypto3/algebra/curves/detail/forms/twisted_edwards/inverted/element_g1.hpp

```diff
@@ -92,10 +92,18 @@ namespace nil {
                          *    @return the selected point (X:Y:Z)
                          *
                          */
-                        constexpr curve_element(const field_value_type& X, const field_value_type& Y, const field_value_type& Z) 
-                            : X(X), Y(Y), Z(Z) 
+                        constexpr curve_element(const field_value_type& X, const field_value_type& Y, const field_value_type& Z)
+                            : X(X), Y(Y), Z(Z)
                         { }
 
+                        /** @brief constructor from affine coordinates
+                         *
+                         */
+                        constexpr curve_element(const field_value_type& X, const field_value_type& Y)
```

Comment:
Do I understand correctly, that for each curve if 2 coordinates were provided, then we assume it's affine coordinates? Because for projective_with_a4_minus_3/element_g1.hpp I can see 

 constexpr curve_element(field_value_type const& X, field_value_type const& Y, field_value_type const& Z = field_value_type::one())
                            : X(X), Y(Y), Z(Z)
Which means if we receive 2 values, it's just setting Z to one().

Reply:
Yes, I changed constructor so if two coordinates are provided it is assumed that they are affine coordinates.
For jacobian and projective coordinates it is just setting to (X:Y:1), but for iverted twisted edwards it is not.
Now all curves accept three or two coordinates.